Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.cs

```diff
@@ -207,10 +209,10 @@ protected override Size ArrangeOverride(Size finalSize)
                 var paddingEnd = new UvMeasure(Orientation, Padding.Right, Padding.Bottom);
                 var position = new UvMeasure(Orientation, Padding.Left, Padding.Top);
 
-                double currentV = 0;
+                var currentRow = new Row();
```

Comment:
I don't like this allocation here. Maybe it makes more sense for the Row class to be a struct?

Reply:
Done